airflow/contrib/operators/gcp_vision_operator.py

```diff
@@ -0,0 +1,603 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from google.api_core.exceptions import AlreadyExists
+
+from airflow.contrib.hooks.gcp_vision_hook import CloudVisionHook
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class CloudVisionProductSetCreateOperator(BaseOperator):
+    """
+    Creates a new ProductSet resource.
+
+    :param product_set: (Required) The ProductSet to create. If a dict is provided, it must be of the same
+        form as the protobuf message `ProductSet`.
+    :type product_set: dict or google.cloud.vision_v1.types.ProductSet
+        https://googleapis.github.io/google-cloud-python/latest/vision/gapic/v1/types.html#google.cloud.vision_v1.types.ProductSet
+    :param location: (Required) The region where the ProductSet should be created. Valid regions are:
+        us-east1, us-west1, europe-west1, asia-east1
```

Comment:
The problem is: that information is not in the official documentation...
I extracted it from the error message from the API. I thought it might be better to give this to the user than give nothing.

Reply:
I think that we should not mention it. I support @potiuk . The difficult art of writing documentation is to write it in such a way that it is always fresh. The documentation should not write about things that are subject to change. Examples: prices, areas of activity.

Reply:
On the other hand we want to be as "user-friendly" as possible. For me (as a user while developing the operators), when testing the Cloud Vision API it was a pain that I couldn't find valid `location`s anywhere in the docs. So I decided to make it easier for the user and include it in the docstring.

I agree that ideally we should be able to link to the Cloud Vision docs, but sadly they don't contain that information. I believe this is better than nothing.

In terms of it being "fresh", as we discussed previously in the case of other operators, it is generally up to the "community" to update the docs when someone notices that it doesn't correspond to the reality anymore.

Also I could add information on when this information was accurate, i.e.:
`Valid regions (as of 2019-02-05) are:`

What do you think? I think it's better than nothing. Also we used to do similar things in the past for other operators, by copying some range of accepted values from the docs, if I recall correctly.

Reply:
Agree. "as of" might be a good idea.

Reply:
Done